### DISCUSSION TITLE
Creating a new project

### DISCUSSION THREAD

**Discussion by JeffreySumner** [posit-dev/positron#5188] [Feedback]
Has anyone else attempted to create a new project in 2024.11.0-116 or 2024.11.0-69? I do not get the pop-up like I used to get on previous versions that asked me to open in current window or new window. I wanted to check here in case this was user error before reporting it as an issue.

**Comment by lionel-** [comment]
hmm I do get the popup after completing the information about the new project.

<img width="349" alt="Screenshot 2024-10-29 at 10 06 08" src="https://github.com/user-attachments/assets/790f0632-2cab-455d-9336-3b3e43c5f7fe">

Could you make a screencast/video of what you're seeing please?

@sharon-wang does this issue ring a bell?

**Reply by sharon-wang** [reply]
@JeffreySumner are you using <kbd>New Project</kbd> or <kbd>New Folder</kbd>/<kbd>New Folder from Git</kbd>?

<img width="239" alt="image" src="https://github.com/user-attachments/assets/02b730f5-267d-4fd2-bafc-b1057c2ffc69">

I'm seeing the same result as @lionel- when I use <kbd>New Project</kbd> to create a new project in 
2024.11.0 build 69. This current/new window popup should show every time, so if that's not happening then we have an issue on our hands!

For <kbd>New Folder</kbd>/<kbd>New Folder from Git</kbd>, there isn't a check for current/new window preference -- could this be what you're seeing?

**Comment by JeffreySumner** [comment]
Apologies for the delay. Here is a test. I uninstalled positron then re-installed the user version and the non-user version in two different locations (4 total tests) and each one comes up with the same result.

https://www.loom.com/share/35c1443d03fd49c0bde5e70b5c18a035?sid=7a28a75b-83cc-443e-a34a-117c811ba7aa

Let me know if there is a loom issue.

I am on windows 11 fyi

**Comment by JeffreySumner** [comment]
This issues does not occur with the same process for v 2024.11.0-49. https://www.loom.com/share/e8d3ee47619e4983b0ff8b75a906a528?sid=ceb31af6-4130-43bc-9465-864d99f505d1

**Comment by JeffreySumner** [comment]
I hadn't tested new folder until now... when testing new folder I get the following error in 2024.11.0-116
![image](https://github.com/user-attachments/assets/c6e0591a-ee7f-4bbe-9bb9-fae0cf83f9a5)

This did not occur in a 2024.11.0-49 test a moment ago

**Comment by sharon-wang** [comment]
Thanks @JeffreySumner this info has been really helpful! I can reproduce on Windows. I'll open an issue for this.

**Comment by JeffreySumner** [comment]
When testing git, I get the following error:
![image](https://github.com/user-attachments/assets/1ebadf37-452b-4c47-8f1b-a1651e41fe85)
This occurs when using the default pathing options. When I adjust the path an I take the /c: off, it works.

I've tried the same for both new folder and new project but those still give the same errors.

Sorry for all the messages, I'm wanting to make sure I test each of the options available.

**Reply by sharon-wang** [reply]
Thank you for the detail, it's so appreciated! I've opened https://github.com/posit-dev/positron/issues/5195. Hope to fix this up soon as it blocks all of these methods of creating projects on Windows :(

**Reply by sharon-wang** [reply]
@JeffreySumner Ah I forgot to update you here! This was fixed in Positron 2024.11.0-140 😄
